topicctl_parser/notifications/datadog.py

```diff
@@ -0,0 +1,37 @@
+import click
+import time
+import httpx
+import os
+
+from typing import Any, Dict
+
+
+DD_API_BASE = "https://api.datadoghq.com"
+
+DATADOG_API_KEY = os.getenv("DATADOG_API_KEY") or os.getenv("DD_API_KEY")
```

Comment:
Please load environment variables inside the CLI script and pass the values as arguments of your method calls. 
This would make your functions self contained and their behavior would only depend on their arguments rather on the state of the system.
This makes them easier to understand and easier to test.
Moreover it would make them usable in contexts where you cannot simply rely on the env var to be set.